src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -911,7 +911,7 @@ static void uct_rc_mlx5_tag_query(uct_rc_mlx5_iface_common_t *iface,
     iface_attr->cap.tag.rndv.max_iov         = 1;
     iface_attr->cap.tag.recv.max_zcopy       = port_attr->max_msg_sz;
     iface_attr->cap.tag.recv.max_iov         = 1;
-    iface_attr->cap.tag.recv.min_recv        = 0;
+    iface_attr->cap.tag.recv.min_recv        = UCT_IB_MLX5_CQE128_MAX_INL;
```

Comment:
need to check current sc2cqe configuration:
if user disabled scatter2cqe, it should be 0